src/mechanisms/hmacsha256.rs

```diff
@@ -18,7 +18,10 @@ impl DeriveKey for super::HmacSha256 {
         if !matches!(key.kind, key::Kind::Symmetric(..) | key::Kind::Shared(..)) {
             // We have to disable this check for compatibility with fido-authenticator, see:
             // https://github.com/solokeys/fido-authenticator/issues/21
-            warn!("derive_key for hmacsha256 called with invalid key kind ({:?})", key.kind);
+            warn!(
+                "derive_key for hmacsha256 called with invalid key kind ({:?})",
+                key.kind
+            );
```

Comment:
This change looks unrelated

Reply:
moved to: https://github.com/trussed-dev/trussed/pull/69

Reply:
It's still here, unless there is something with my GH view.

Reply:
Yes, this PR has not been rebased yet.  But if you compare to current main, there are no changes to `src/mechanisms/hmacsha256.rs` anymore.

Reply:
Right. Resolving.